storypoint/storypoint/CollectionViewCellThemeHelper.swift

```diff
@@ -48,5 +48,14 @@ class CollectionViewCellThemeHelper {
             cell.titleLabel.font =  UIFont.systemFont(ofSize: (item as! SelectedStoryPointCard).fontSize)
         }
     }
+}
+
+extension UIImage {
+    static func randomPony() -> UIImage? {
+        let ponies = ["apple.jpg", "pinkie.jpg", "pinkie2.png", "rarity.png", "rarity.jpg", "applejack.png", "dash.png", "dash2.png", "dash3.png", "maud.png", "scootaloo.png", "twilight.png", "twilight2.png"]
+        
+        let index = Int(arc4random()) % ponies.count
```

Comment:
`arc4random` is pretty wack sounding, isn't it? Is it just like Math.random() in C#? This will work just fine, but I _think_ the general consensus these days is to use [`arc4random_uniform()`](https://stackoverflow.com/questions/17640624/arc4random-modulo-biased). You could take a look if you're curious